cmd/run/run_test.go

```diff
@@ -612,13 +612,15 @@ func TestBuildServerWithOIDCAuthenticationAlias(t *testing.T) {
 	oidcServerPort2, oidcServerPortReleaser2 := testutils.TCPRandomPort()
 	oidcServerURL1 := fmt.Sprintf("http://localhost:%d", oidcServerPort1)
 	oidcServerURL2 := fmt.Sprintf("http://localhost:%d", oidcServerPort2)
+	subject := "some-user"
 
 	cfg := testutils.MustDefaultConfigWithRandomPorts()
 	cfg.Authn.Method = "oidc"
 	cfg.Authn.AuthnOIDCConfig = &serverconfig.AuthnOIDCConfig{
 		Audience:      "openfga.dev",
 		Issuer:        oidcServerURL1,
 		IssuerAliases: []string{oidcServerURL2},
+		Subjects:      []string{subject},
```

Comment:
You don't need this line. If `subjects` is empty, every subject is allowed so this test will still pass. I wouldn't update this entire test at all since it's testing something else.

But we should add to the description of the new flag that if subjects is empty, every subject is allowed